pkg/proxy/iptables/proxier_test.go

```diff
@@ -709,6 +711,87 @@ func errorf(msg string, rules []iptablestest.Rule, t *testing.T) {
 	t.Errorf("%v", msg)
 }
 
+// findAllMatches takes an array of lines and a pattern with one parenthesized group, and
+// returns a sorted array of all of the unique matches of the parenthesized group.
```

Comment:
this is sorted by string size, is this the expectation?

Reply:
ic, so you aggregate the lines by the regex, if you match the service name it aggregates by service
you can close this comment

Reply:
no it's not? We insert all the matches into a string set, and then `allMatches.List()` returns the elements sorted as strings...

Reply:
yeah, I wrote a nonsense, but I got it after checking how the test uses this function, :sweat_smile: